File: src/ast/PostfixQualifiedType.java
package ast;

import java.io.StringWriter;

public class PostfixQualifiedType extends ParentNode {
    private String _postfixQualifier;

    public PostfixQualifiedType(String postfixQualifier, BaseNode type) {
        super(NodeType.PostfixQualifiedType, type);
        _postfixQualifier = postfixQualifier;
    }

    @Override
    public void PrintLeft(StringWriter writer) {
        child.Print(writer);
        writer.write(_postfixQualifier);
    }
}

File: src/de/fearlesstobi/demangler/ast/PostfixQualifiedType.kt
package de.fearlesstobi.demangler.ast

import java.io.StringWriter

class PostfixQualifiedType(private val postfixQualifier: String, type: BaseNode?) : ParentNode(NodeType.PostfixQualifiedType, type) {
    public override fun printLeft(writer: StringWriter) {
        child!!.print(writer)
        writer.write(postfixQualifier)
    }

}